src/autopas/LogicHandler.h

```diff
@@ -181,23 +181,32 @@ class LogicHandler {
   /**
    * @copydoc AutoPas::reserve()
    *
-   * Reserves space in the particle buffers.
+   * Reserves space in the particle buffers and the container. Estimates number of halo particles.
    */
   void reserve(size_t numParticles) {
     const auto &container = _autoTuner.getContainer();
     const auto numParticlesHaloEstimate = autopas::utils::NumParticlesEstimator::estimateNumHalosUniform(
         numParticles, container->getBoxMin(), container->getBoxMax(), container->getInteractionLength());
-    const auto numParticlesHaloEstimatePerBuffer = numParticlesHaloEstimate / _haloParticleBuffer.size();
+    reserve(numParticles, numParticlesHaloEstimate);
+  }
 
+  /**
+   * Reserves space in the particle buffers and the container.
```

Comment:
Similar to above - wouldn't this be more suitable with a copydoc to avoid duplication

Reply:
7fc9847